src/App.tsx

```diff
@@ -34,10 +36,15 @@ function App(): React.ReactElement {
         <nav>
           <ul className={styles.navList}>
             <li className={styles.navItem}>
-              <Link to="/" style={{ textDecoration: 'none' }}>
+              <Link to="/" className={styles.button}>
```

Comment:
coloca um botão entre o link. Não faz sentido utilizar o estilo `button` em um `link`
```suggestion
              <Link to="/" className={styles.button}>
                    <Button> Todas as lojas </Button>
              </Link>                          
```